src/main/kotlin/org/jitsi/jibri/sipgateway/pjsua/PjsuaClient.kt

```diff
@@ -70,14 +76,31 @@ class PjsuaClient(
             PJSUA_SCRIPT_FILE_LOCATION
         )
 
+        /**
+         * A SIP address is written in user@domain.tld format in a similar fashion to an email address.
+         * An address like: sip:1-999-123-4567@voip-provider.example.net
+         *
+         * The default sip scheme is `sip` if none is specified; Valid options would be `sip` or `sips`
+         */
+        var sipScheme = "sip"
+        var sipAddress = pjsuaClientParams.sipClientParams.sipAddress
+        if (sipAddress.isNotEmpty() && sipAddress.hasSipSchemeEmbedded()) {
```

Comment:
A small nit, but I might do this as:

create a `String.getSipScheme(): String?` that returns either the scheme or null, and then you can do:

`val sipScheme = pjsuaCLientParams.sipClientParams.getSipScheme() ?: "sip"`

And then also create a `String.getSipAddress(): String` that checks if the scheme is there and does the appropriate thing so then you get.

`val sipAddress = pjsuaCLientParams.sipClientParams.sipAddress.getSipAddress()`

Not a huge deal, but just a thought (it gets rid of the vars and the needs for assigning twice).